tools/fast/bin/localnet/main.go

```diff
@@ -66,8 +71,27 @@ func init() {
 		minerExpiryArg     = minerExpiry.Text(10)
 	)
 
+	// We default to the binary built in the project directory, fallback
+	// to searching path.
+	binpath, err = getFilecoinBinary()
+	if err != nil {
+		// Look for `go-filecoin` in the path to set `binpath` default
+		// If the binary is not found, an error will be returned. If the
+		// error is ErrNotFound we ignore it.
+		// Error is handled after flag parsing so help can be shown without
+		// erroring first
+		binpath, err = exec.LookPath("go-filecoin")
+		if err != nil {
+			xerr, ok := err.(*exec.Error)
+			if ok && xerr.Err == exec.ErrNotFound {
```

Comment:
if not ok bail?

Reply:
No, we only want to check the second half of the expression if the cast was successful.